CRM/ManualDirectDebit/Hook/PageRun/CustomGroupInjector.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/**
+ *  Provides 'Direct Debit Information' integration into 'Recurring Contribution Detail' view
+ */
+class CRM_ManualDirectDebit_Hook_PageRun_CustomGroupInjector {
```

Comment:
I think better to rename thsi class to : 

CRM_ManualDirectDebit_Hook_PageRun_ContributionRecur_DirectDebitFieldsInjector

and rename injectDirectDebitInformation() to inject()